//
// Student License - for use by students to meet course requirements and
// perform academic research at degree granting institutions only.  Not
// for government, commercial, or other organizational use.
//
// abcefg_TA_AVP.h
//
// Code generation for function 'abcefg_TA_AVP'
//

#ifndef ABCEFG_TA_AVP_H
#define ABCEFG_TA_AVP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abcefg_TA_AVP(double P_init, double V_init, double A_init, double P_wayp,
                   double V_wayp, double A_wayp, double A_max, double A_min,
                   double J_max, double J_min, double T, creal_T t[28]);

#endif
// End of code generation (abcefg_TA_AVP.h)
